### ISSUE TITLE
Configuration file with glove embeddings

### ISSUE THREAD

**Issue by mdelhoneux** [DreamerDeo/HIT-SCIR-CoNLL2019#8]
Hi,

The paper reports results with a baseline that uses GloVe embeddings instead of BERT, but unless I'm missing something, I don't see the configuration files (+training instructions) for that? Do you still have that somewhere and could you upload it to the repo?

**Comment by longxudou** [comment]
Hi, 

I have push the glove config and corresponding training bash. Please have a check.
Most of hyper-parameters are the same as the bert config. Some hyper-parameters could be tuned for better performance, i.e dropout and learning rate.

**Comment by mdelhoneux** [comment]
Great, thanks! One minor thing is that there's a mismatch between CUDA_VISIBLE_DEVICES in the bash files which are called CUDA_DEVICE in the config files. The rest looks good and seems functional :)
